avalon/tools/cbloader/widgets.py

```diff
@@ -240,6 +242,8 @@ def set_version(self, version_id):
 
         self.setEnabled(True)
 
+        print("Querying..")
```

Comment:
Was this for debugging purposes, or is this a logical message to display for the end user?

Reply:
Sure - we can keep it. If it's too verbose later on we can always tone it down.